src/languages/python/uv.rs

```diff
@@ -221,32 +222,64 @@ impl Uv {
         Ok(source)
     }
 
+    async fn get_uv_version(uv_path: &Path) -> Result<Version> {
+        let output = Cmd::new(uv_path, "Checking uv version")
+            .arg("--version")
+            .check(false)
+            .output()
+            .await?;
+
+        if !output.status.success() {
+            bail!("Failed to get uv version");
+        }
+
+        let version_output = String::from_utf8_lossy(&output.stdout);
+        let version_str = version_output
+            .split_whitespace()
+            .nth(1)
+            .ok_or_else(|| anyhow::anyhow!("Invalid version output format"))?;
+
+        Version::parse(version_str).map_err(Into::into)
+    }
+
     pub async fn install(store: &Store) -> Result<Self> {
-        // 1) Check if `uv` is installed already.
-        // TODO: check minimum supported uv version
-        if let Ok(uv) = UV_EXE.as_ref() {
-            return Ok(Self::new(uv.clone()));
+        // 1) Check if system `uv` meets minimum version requirement
+        if let Ok(uv_path) = UV_EXE.as_ref() {
+            if let Ok(version) = Self::get_uv_version(uv_path).await {
```

Comment:
I also want to move the quering version logic into the `UV_EXE` LazyLock, so we don't need to query multiple times. And change from `which::which("uv")` to `which::which_all("uv")` to find all system uv.